### PR TITLE
gradle: adds additional gradle examples

### PR DESCRIPTION
**Pull request by bavaga**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/gradle.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
 
 - Compile a package:
 
 `gradle build`
 
+- Exclude test task:
+
+`gradle build -x test`
+
+- Run in offline mode:
```
Comment (agnivade) [inline pages/common/gradle.md]: Can you expand the description a bit to explain what this offline mode is ? It is not immediately clear to me what it does.

File: pages/common/gradle.md [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
-> Gradle is the official build system for Android Studio.
+> Gradle is an advanced open source build automation system.
 
 - Compile a package:
 
 `gradle build`
 
+- Exclude test task:
+
+`gradle build -x test`
```
Comment (agnivade) [inline pages/common/gradle.md]: Shouldn't `test` be inside tokens (`{{}}`) ?

File: pages/common/gradle.md [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
+
 - Clear the build folder:
 
 `gradle clean`
 
 - Compile and Release package:
 
 `gradle assembleRelease`
+
+- Create Gradle wrapper:
```
Comment (agnivade) [inline pages/common/gradle.md]: We have to also explain the options used in the command. Either change the description to `Create gradle wrapper with a given gradle version`, or change the example to exclude specifying the gradle-version. 

Remember to always focus on the most common and practical examples. If specifying the gradle-version is not a common thing, then we can drop it.

**Comment by agnivade** [comment]
I see that you had closed a previous PR and opened a new one. That is not necessary. You can update the same PR again by pushing new commits.

File: pages/common/gradle.md (hunk 2 above)
Comment (bavaga) [reply pages/common/gradle.md]: Thanks. Done.

File: pages/common/gradle.md (hunk 1 above)
Comment (bavaga) [reply pages/common/gradle.md]: 👍

File: pages/common/gradle.md (hunk 3 above)
Comment (bavaga) [reply pages/common/gradle.md]: I've removed that example. It seems not to be a very common use case.

**Comment by bavaga** [comment]
Thank you for the feedback. I integerated it in the new commit.

**Comment by sbrl** [comment]
Looks ok to me! Thanks, @bavaga 😺

**Pull request merged**